plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java

```diff
@@ -282,6 +283,41 @@ else if (attribute.startsWith("water", 2)) {
             }
             return list;
         });
+
+        // <--[tag]
+        // @attribute <BiomeTag.foliage_color>
+        // @returns ColorTag
+        // @mechanism BiomeTag.foliage_color
+        // @description
+        // Returns the foliage color of this biome. Foliage includes leaves and vines.
+        // If the biome's foliage does not change color by default, this returns the color black.
+        // -->
+        tagProcessor.registerTag(ColorTag.class, "foliage_color", (attribute, object) -> {
+            return new ColorTag(ColorTag.fromRGB(object.biome.getFoliageColor()));
+        });
+
+        // <--[mechanism]
+        // @object BiomeTag
+        // @name foliage_color
+        // @input ColorTag
+        // @description
+        // Sets the foliage color of this biome. Foliage includes leaves and vines.
+        // Can be changed once until next server restart. Colors reset on server restart.
+        // For the change to take effect on the players' clients, they must quit and rejoin the server.
+        // @tags
+        // <BiomeTag.foliage_color>
+        // @example
+        // # Adjusts the foliage color of the plains biome permanently, using a server start event to keep it applied.
+        // # Now the leaves and vines will be a nice salmon-pink!
+        // on server start:
+        // - adjust <biome[plains]> foliage_color:<&color[#F48D8D]>

```

Comment:
Arrrrre you sure that's correct? `&color` is for chat colors, not `ColorTag` - technically you'd use `ColorTag`'s constructor but in this case can probably input the hex code directly as it's a valid `ColorTag`.